alacritty/src/display/mod.rs

```diff
@@ -771,6 +778,46 @@ impl Display {
         });
     }
 
+    /// Draw the hyperlink target URI preview of which are currently active.
+    fn draw_active_hyperlink_preview(&mut self, config: &Config, size_info: &SizeInfo) {
+        // Is there really anyone hovering two links simontinously?
```

Comment:
Well is it possible? Each cell should only have one hyperlink, so we shouldn't have to consider this.

Reply:
Actually yes. You can go vi mode and put your cursor over one, and move your mouse to point another one.

Reply:
That's definitely possible and should already work with normal hints. The both should be treated independently without any issues though. "Hovering" as in hovering above with the mouse should only be possible with one.